presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -489,6 +489,17 @@ public void testInsert()
                 "key3", null, 999, null, null, null, "ansi", false, null, null, null, null, null, null, null, null, null, null));
     }
 
+    @Test
+    public void testSelectInformationSchemaColumns()
+    {
+        assertQuery(
+                "SELECT table_schema FROM information_schema.columns WHERE table_schema = 'tpch' LIMIT 1",
```

Comment:
use `ORDER BY` to make query be deterministic

Reply:
FYI, The columns in SELECT and WHERE are same, so I eliminated ORDER BY intentionally.